pkg/util/env.go

```diff
@@ -0,0 +1,74 @@
+/*
+Copyright 2020 Red Hat
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
```

Comment:
this is a dupe of https://github.com/openstack-k8s-operators/osp-director-operator/blob/master/pkg/common/env.go